system/ee/legacy/errors/error_exception.php

```diff
@@ -1,8 +1,8 @@
 <!doctype html>
 <html>
 	<head>
-		<title>Exception - ExpressionEngine</title>
-		<meta http-equiv="content-type" content="text/html; charset=utf-8" lang="en-us" dir="ltr">
+		<title>Exception | <?=APP_NAME?></title>
```

Comment:
This change seems unrelated to adding date formats.

Reply:
Sorry, I don't know why GitHub Desktop SW merged all my pull requests into one.  Please ignore this pull request. I will create another separated.